### PR TITLE
Remove runtime in validate

### PR DESCRIPTION
**Pull request by yanghesong**
Validate is useless as dockershim is removed

Signed-off-by: yanghesong <hesong.yang@foxmail.com>

#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Dockershim has been removed, refer to #97252. Change the corresponding error message of dockershim.
#### Which issue(s) this PR fixes:
Validate is useless as dockershim is removed

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/security/apparmor/validate.go [hunk 1]
```diff
@@ -106,11 +105,6 @@ func validateHost(runtime string) error {
 		return errors.New("AppArmor is not enabled on the host")
 	}
 
-	// Check runtime support. Currently only Docker is supported.
-	if runtime != kubetypes.DockerContainerRuntime && runtime != kubetypes.RemoteContainerRuntime {
-		return fmt.Errorf("AppArmor is only enabled for 'docker' and 'remote' runtimes. Found: %q", runtime)
-	}
-
```
Comment (wzshiming) [inline pkg/security/apparmor/validate.go:113]: Maybe we should keep it, check the case where runtime is not remote

Comment (yanghesong) [reply pkg/security/apparmor/validate.go:113]: image
refer to #107299

**Review by adisky — COMMENTED** [review]
/lgtm

**Review by mrunalp — APPROVED** [review]

**Review by wzshiming — APPROVED** [review]

**Comment by dims** [comment]
@adisky please let me know when this is ready.

**Comment by adisky** [comment]
this is ready to go @dims

**Comment by dims** [comment]
/approve

NOTE: cleanup after dockershim

**Pull request merged**